components/webhooks/cmd/serve.go

```diff
@@ -1,30 +1,68 @@
 package cmd
 
 import (
+	"strings"
 	"syscall"
+	"time"
 
 	"github.com/formancehq/stack/libs/go-libs/logging"
 	"github.com/formancehq/stack/libs/go-libs/service"
 	"github.com/formancehq/webhooks/cmd/flag"
 	"github.com/formancehq/webhooks/pkg/otlp"
 	"github.com/formancehq/webhooks/pkg/server"
+	"github.com/formancehq/webhooks/pkg/storage/postgres"
+	"github.com/formancehq/webhooks/pkg/worker"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"go.uber.org/fx"
 )
 
-var serverCmd = &cobra.Command{
-	Use:   "serve",
-	Short: "Run webhooks server",
-	RunE:  RunServer,
+func newServeCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "serve",
+		Short: "Run webhooks server",
+		RunE:  serve,
+	}
 }
 
-func RunServer(cmd *cobra.Command, _ []string) error {
+func serve(cmd *cobra.Command, _ []string) error {
 	logging.FromContext(cmd.Context()).Debugf(
-		"starting webhooks server module: env variables: %+v viper keys: %+v",
-		syscall.Environ(), viper.AllKeys())
+		"starting webhooks server module: env variables: %+v viper settings: %+v",
```

Comment:
I know this was already there, but can you remove it?
It leaks some secrets on logs (which are sent to uptraces).

Reply:
I removed it